pkg/conditions/replicasready/update.go

```diff
@@ -0,0 +1,47 @@
+package replicasready
+
+import (
+	capi "sigs.k8s.io/cluster-api/api/v1alpha3"
+	capiexp "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
+	capiconditions "sigs.k8s.io/cluster-api/util/conditions"
+)
+
+func update(machinePool *capiexp.MachinePool) {
```

Comment:
Minor nitpick: Maybe this function name could be slightly more explicit like `alignStatus` or `alignStatusConditions` or `alignConditions` ?

Reply:
Yeah, Chris had similar comment, I would prefer to do those changes in a separate PR, see this comment https://github.com/giantswarm/conditions-handler/pull/14#discussion_r548538165